python/ray/air/util/node.py

```diff
@@ -0,0 +1,36 @@
+import ray
+
+
+def _get_node_id_from_node_ip(node_ip: str):
+    """Returns the node ID for the first alive node with the input IP."""
+    for node in ray.nodes():
+        if node["Alive"] and node["NodeManagerAddress"] == node_ip:
+            return node["NodeID"]
+
+    return None
+
+
+def _force_on_current_node(task_or_actor=None):
```

Comment:
```suggestion
def _force_on_current_node(task_or_actor: Optional[Union[ray.ObjectRef, ray.actor.ActorClass]] = None):
```

Reply:
Updated, though I believe the first should actually be a `ray.remote_function.RemoteFunction` - updated the arg name for clarity as well.

Reply:
Ah sorry, you're right!